sources.list

```diff
@@ -1,27 +1,27 @@
 # See http://help.ubuntu.com/community/UpgradeNotes for how to upgrade to
 # newer versions of the distribution.
 
-deb http://archive.ubuntu.com/ubuntu/ trusty main restricted
-# deb-src http://archive.ubuntu.com/ubuntu/ trusty restricted main multiverse universe
+deb http://archive.ubuntu.com/ubuntu/ xenial main restricted
```

Comment:
@yukiisbored just delete of your comments or I would have to approve this that means the next one too! Sorry I tried to delete the entire review but only my comment vanished!